File: ui-admin/src/app/backend/payloads/business/caisse-session-payload.ts
export class CaisseSessionPayload {
    id: number;
    openedAt: string;
    closedAt: string;
    openingBalance: number;
    closingBalance: number;
    totalSalesAmount: number;
    totalSalesCount: number;
    firstOrderNumber: number;
    lastOrderNumber: number;
    variance: number;
    status: string;
    notes: string;
    posId: number;
    posCode: string;
    posName: string;
    createdByFullName: string;
    createdAt: string;
    updatedAt: string;

    constructor() {
        this.id = null;
        this.openedAt = null;
        this.closedAt = null;
        this.openingBalance = null;
        this.closingBalance = null;
        this.totalSalesAmount = null;
        this.totalSalesCount = null;
        this.firstOrderNumber = null;
        this.lastOrderNumber = null;
        this.variance = null;
        this.status = null;
        this.notes = null;
        this.posId = null;
        this.posCode = null;
        this.posName = null;
        this.createdByFullName = null;
        this.createdAt = null;
        this.updatedAt = null;
    }
}

File: api/src/main/java/com/mystore/manager/api/business/payload/CaisseSessionPayload.java
package com.mystore.manager.api.business.payload;

import com.mystore.manager.api.common.payload.GlobalUserDatePayload;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class CaisseSessionPayload extends GlobalUserDatePayload {
    private String openedAt;
    private String closedAt;
    private Double openingBalance;
    private Double closingBalance;
    private Double totalSalesAmount;
    private Integer totalSalesCount;
    private Integer firstOrderNumber;
    private Integer lastOrderNumber;
    private Double variance;
    private String status;
    private String notes;
}
